tests/cython/README.md

```diff
@@ -1,27 +1,33 @@
 
-HOWTO run these tests by hand:
-------------------------------
+## Preconditions
 
-You need to set up the PYTHON path:
+```nose``` Python testing framework is required.
```

Comment:
fwiw -- I've been installing with `apt-get install nose` -- I also have both nose and nose3 isntalled; I'm not sure that is even possible with pip and conda.